### PR TITLE
hub dataset to geojson feature

### PR DESCRIPTION
**Pull request by sansth1010**
PR for [#5619](https://devtopia.esri.com/dc/hub/issues/5619)

### CODE REVIEW

File: src/helpers/enrich-dataset.ts [hunk 1]
```diff
@@ -55,12 +54,35 @@ export function enrichDataset(dataset: HubDataset, hubsite: HubSite) {
...
+function hubDatasetToFeature(hubDataset: HubDataset) {
+    const { type, rings } = hubDataset?.boundary?.geometry ?? {};
+
+    return {
+        type: 'Feature',
+        geometry: {
+            type: type && capitalize(type),
+            coordinates: rings
+        },
+        properties: _.omit(hubDataset, ['boundary'])
```
Comment (rgwozdz) [inline src/helpers/enrich-dataset.ts:72]: I've read that `omit` is inefficient.  Check this post for alternatives: https://github.com/FormidableLabs/victory/issues/956

File: src/helpers/enrich-dataset.ts [hunk 2]
```diff
@@ -55,12 +54,35 @@ export function enrichDataset(dataset: HubDataset, hubsite: HubSite) {
...
+}
+
+function capitalize(word) {
+    return word
+        .split('')
+        .map((letter, index) =>
+            index ? letter.toLowerCase() : letter.toUpperCase(),
+        )
+        .join('');
+}
```
Comment (rgwozdz) [inline src/helpers/enrich-dataset.ts:83]: Maybe more efficient to do a lookup with an object?

`{ polygon: 'Polygon' }`

We should only ever have polygon in the index, but if you want to future proof: https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html#input-structure

I am a little surprise the search API doesn't transform this for us before we get here.  I would ask Tom Wayson if the current format delivered by the search API should be altered.

**Review by rgwozdz — CHANGES_REQUESTED** [review]

File: src/helpers/enrich-dataset.ts (hunk 1 above)
Comment (sansth1010) [reply src/helpers/enrich-dataset.ts]: Used babel transpiler method to remove key from an object. Considered to be the fastest.

File: src/helpers/enrich-dataset.ts (hunk 2 above)
Comment (sansth1010) [reply src/helpers/enrich-dataset.ts]: Used look up for mapping elastic type to GeoJSON geometry type.

**Review by rgwozdz — APPROVED** [review]
Awesome, nice work.

**Pull request merged**